bindings/Sofa/src/SofaPython3/Sofa/Core/Binding_Node.cpp

```diff
@@ -360,6 +362,52 @@ p.def("__getitem__", [](Node& self, const std::string& s) -> py::object
     return getItem(self, stringlist);
 
 });
+p.def("removeObject", &Node::removeObject, sofapython3::doc::sofa::core::Node::removeObject);
+p.def("getRootPath", &Node::getRootPath, sofapython3::doc::sofa::core::Node::getRootPath);
+p.def("moveChild", [](Node *self, BaseNode* child, BaseNode* prevParent){
+    if (child && prevParent){
+        self->moveChild(child, prevParent);
+        return;
+    }
+    throw py::value_error("Invalid arguments provided");
+}, sofapython3::doc::sofa::core::Node::moveChild);
+p.def("isInitialized", &Node::isInitialized, sofapython3::doc::sofa::core::Node::isInitialized);
+p.def("getAsACreateObjectParameter", [](Node *self){
+    return getLink(self);
+}, sofapython3::doc::sofa::core::Node::getAsACreateObjectParameter);
+p.def("detachFromGraph", &Node::detachFromGraph, sofapython3::doc::sofa::core::Node::detachFromGraph);
+p.def("getMass", [](Node *self) ->py::object {
+    sofa::core::behavior::BaseMass* mass = self->mass.get();
+    if (mass){
+        return PythonDownCast::toPython(mass);
+    }
+    return py::none();
+}, sofapython3::doc::sofa::core::Node::getMass);
+p.def("getForceField", [](Node *self, unsigned int index) ->py::object{
+    sofa::core::behavior::BaseForceField* ff = self->forceField.get(index);
+    if (ff){
+        return PythonDownCast::toPython(ff);
+    }
+    return py::none();
+}, sofapython3::doc::sofa::core::Node::getForceField);
+p.def("getMechanicalState", [](Node *self) -> py::object{
+    sofa::core::behavior::BaseMechanicalState* state = self->mechanicalState.get();
+    if (state){
+        return PythonDownCast::toPython(state);
+    }
+    return py::none();
+}, sofapython3::doc::sofa::core::Node::getMechanicalState);
+p.def("getMechanicalMapping", [](Node *self) ->py::object{
+    sofa::core::BaseMapping* mapping = self->mechanicalMapping.get();
+    if (mapping){
+        return PythonDownCast::toPython(mapping);
+    }
+    return py::none();
+}, sofapython3::doc::sofa::core::Node::getMechanicalMapping);
+p.def("sendMessage", [](Node* self, py::object* pyUserData, char* eventName){
```

Comment:
Hello, I was wrong. To make it consistant with the name of PythonScriptEvent it is better
to name that sendEvent. (Sorry)